### DISCUSSION TITLE
TSML UI District?

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#826] [Q&A] [answered]
I don't believe I've seen this before. Where is it supposed to pull the data from?

We have several meetings with group information that includes districts, yet as it's displayed on the meeting grid, it's blank.

I am using v3.14.4 

Switching to Legacy does not have a column for district.

**Comment by joshreisner** [comment, marked as answer]
TSML UI displays the district when present on the meeting detail page but not in the list. Here is some history:

* https://github.com/code4recovery/tsml-ui/issues/125
* https://github.com/code4recovery/tsml-ui/issues/106
* https://github.com/code4recovery/12-step-meeting-list/discussions/762#discussioncomment-2568747

**Reply by Grateful-Viv** [reply]
It doesn’t display any districts even though the column header is there.

**Reply by joshreisner** [reply]
oops i'll fix that

**Reply by joshreisner** [reply]
should be fixed now

**Reply by Grateful-Viv** [reply]
Yes, District is no longer a column. Was that what you intended? It's okay with me not to show it on the list page. And yes, it District info (if there is any) does show up on the detail page.

Thank you.

**Reply by joshreisner** [reply]
yes, i didn't mean for district to show up as a column. my goal is that it be represented on the detail page and that it be searchable.

because it's searchable, sites that want to (such as area websites) could create a custom navigation in their top menu:

**District**
* District 01 &rarr; `/meetings?search="district+01"`
* District 02 &rarr; `/meetings?search="district+02"`

**Reply by Grateful-Viv** [reply]
So in effect this replaces the switch to districts/regions option that used to be. It’s great! Thank you. We can consider this question answered and closed.
